.github/workflows/cd-build-push.yml

```diff
@@ -0,0 +1,58 @@
+name: "CD Build and Push"
```

Comment:
It is not a Push but rather an on push. According to the other build names, this should only be "CD Build".

Take a look at https://github.com/remsfal/remsfal-backend/actions where the workflow names are displayed.